pom.xml

```diff
@@ -289,6 +289,20 @@
             <plugins.spymemcached>spymemcached</plugins.spymemcached>
          </properties>
       </profile>
+      <profile>
```

Comment:
It would be nice if you could trigger plugin compilation by the fact that another plugin that has this as a dependency is compiled. But I don't know if that could be done in any easy way (I think that properties defined in profile cannot activate another profiles).